lib/client-connection.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * firebase-server 0.4.0
+ * License: MIT.
+ * Copyright (C) 2013, 2014, 2015, Uri Shaked.
+ * Copyright (C) 2015 James Talmage.
+ */
+
+'use strict';
+
+var util = require('util');
+var EventEmitter = require('events').EventEmitter;
+var Request = require('./Request');
+var debug = require('debug')('firebase-server:client-connection');
+
+module.exports = ClientConnection;
+
+function ClientConnection(ws, server) {
```

Comment:
I am only using ECMASCRIPT 6 in the tests.

See the `ignore` option in `test/mocha-babel.js`.

I hunted around and this seemed to be the best strategy. I've been working the AVA team recently (which is pretty Babel centric testing framework), so I may have some improvements to make when I get back into this.